src/types/ape-text-field.d.ts

```diff
@@ -0,0 +1,16 @@
+import { TextFieldProps } from '@material-ui/core';
```

Comment:
I'd have this in the same ApeTextField.tsx

When i've needed to share these types of things, I put them in `global.d.ts`

Can we keep this in `ApeTextField.tsx` and the decide how to structure this in the future. I think if we had a seperate file we'd want it in the same component folder with storybook, ect.

Like I said, I'd like us to have that type of conversation in another PR that isn't blocking, and come to a decision with a few engineers about what we want for our standards.

Reply:
I just removed from the ApeTextField.tsx since aI using one combined type

Reply:
This is the only thing blocking me approving.